apps/native/app/src/components/pdf-viewer/pdf-viewer.tsx

```diff
@@ -0,0 +1,63 @@
+import { DdLogs } from '@datadog/mobile-react-native'
+import { memo, useState } from 'react'
+import { Platform } from 'react-native'
+import Pdf from 'react-native-pdf'
+
+interface PdfViewerProps {
+  url: string
+  subject: string
+  senderName: string
+  onLoaded(path: string): void
+  onError(err: Error): void
+}
+
+export const PdfViewer = memo(
+  ({ url, subject, senderName, onLoaded, onError }: PdfViewerProps) => {
+    const [actualUrl, setActualUrl] = useState(url)
+    const extraProps = {
+      activityIndicatorProps: {
+        color: '#0061ff',
+        progressTintColor: '#ccdfff',
+      },
+    }
+
+    return (
+      <Pdf
+        spacing={0}
```

Comment:
Almost exactly the same component but moved to its own file. The only difference is spacing 0 property for the pdf to expand fully to its container without shadow